web_tree_many2one_clickable/__openerp__.py

```diff
@@ -1,38 +1,25 @@
 # -*- coding: utf-8 -*-
-##############################################################################
-#
-#    OpenERP, Open Source Management Solution
-#    This module copyright
-#      (C) 2013 Therp BV (<http://therp.nl>).
-#      (c) 2015 Serv. Tecnol. Avanzados (http://www.serviciosbaeza.com)
-#               Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
-#
-#    This program is free software: you can redistribute it and/or modify
-#    it under the terms of the GNU Affero General Public License as
-#    published by the Free Software Foundation, either version 3 of the
-#    License, or (at your option) any later version.
-#
-#    This program is distributed in the hope that it will be useful,
-#    but WITHOUT ANY WARRANTY; without even the implied warranty of
-#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-#    GNU Affero General Public License for more details.
-#
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# Copyright 2013 Therp BV (<http://therp.nl>).
+# Copyright 2015 Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+# Copyright 2015 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
 {
     "name": "Clickable many2one fields for tree views",
-    "version": "8.0.1.0.0",
+    "summary": "Open the linked resource when clicking on their name",
+    "version": "9.0.1.0.0",
+    "category": "Dependency",
```

Comment:
I don't really have an opinion about this, but Hidden/Dependency is what I use a lot for this kind of modules